web/libs/datamanager/src/components/Common/SkeletonLoader/SkeletonLoader.scss

```diff
@@ -88,7 +91,7 @@
 }
 
 
-@-o-keyframes skeleton-animation {
+@keyframes skeleton-animation {
```

Comment:
with linting clearing out the browser specific definitions they became duplicate declarations - we can probably safely remove 3 out of the 4

Reply:
makes sense